android/settings_aar.gradle

```diff
@@ -0,0 +1 @@
+include ':app'
```

Comment:
What is this for?

Reply:
I dont't know :)). It was generated when I built the app.  I will delete it.